.github/workflows/pocket-ic-build-windows.yml

```diff
@@ -0,0 +1,24 @@
+name: 'PocketIC library build on Windows'
+on:
+  pull_request:
+    paths:
+      - packages/pocket-ic/**
+# runs for the same workflow are cancelled on PRs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+permissions: read-all
```

Comment:
is this necessary?

Reply:
I took this from the `ci-main.yml`, but happy to drop it if this is not necessary.

Reply:
It shouldn't be necessary, we should remove this from ci-main to avoid confusion.

Reply:
I've removed the permissions from the new CI flow, but kept in ci-main as far as this PR is concerned.